### PR TITLE
[Client update] August 2022

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Handles client update August 2022

- Titles
- New zones, zone renames, db entries etc.
- New status effect numbers
- Text IDs

## Steps to test these changes

N/A

### CODE REVIEW

**Comment by zach2good** [comment]
Holding on clay getting time to do the mob/npc ID shifts

**Comment by zach2good** [comment]
Will rebae this onto `main` when https://github.com/LandSandBoat/server/pull/2490 lands

File: src/map/zone.h [hunk 1]
```diff
@@ -318,7 +318,7 @@ enum ZONEID : uint16
     ZONE_DHO_GATES                      = 272,
     ZONE_WOH_GATES                      = 273,
     ZONE_OUTER_RAKAZNAR                 = 274,
-    ZONE_OUTER_RAKAZNAR_U               = 275,
+    ZONE_OUTER_RAKAZNAR_U1              = 275,
```
Comment (TeoTwawki) [inline src/map/zone.h:321]: don't forget Throne Room [V] at 229, you got in the lua side but not the cpp header

**Comment by claywar** [comment]
Updated.  NPC causing issues on start was 17727647, which while unrelated to transports in the database, was somehow getting referenced.

**Comment by claywar** [comment]
Future verification needed regarding navmeshes and the renamed zones:
image

**Review by claywar — APPROVED** [review]

**Pull request merged**